### PR TITLE
Task 3 basic grid

### PR DESCRIPTION
**Pull request by Madesine**
Basic grid for project

### CODE REVIEW

File: fonts.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@font-face {
```
Comment (universezzz) [inline fonts.css]: Create folders for your css/js files
css
-- index-page.css (rename index.css)
-- catalog.css 
js
-- script.js
and html files without any folder, just leave it in root foler.
and rename html files
nerds-catalog.html -> catalog.html
nerds-index.html -> index.html
1) First of all, your root folder has name 'nerds bla bla', for what have you used nerds prefix? I don't know
2) Basically, the main html file has 'index.html' name.

File: nerds-catalog.html [hunk 2]
```diff
@@ -9,38 +9,43 @@
...
-          <li><a href="#">Студия</a></li>
-          <li><a href="#">Клиенты</a></li>
-          <li><a href="../Nerds.web/nerds-catalog.html">Магазин</a></li>
-          <li><a href="#">Контакты</a></li>
-        </ul>
-      </nav>
+    <header class="main-header">
+      <div class="header-container">
+        <div class="header-logo">
+          <a href="../Nerds.web/nerds-index.html" aria-label="Лого">
```
Comment (universezzz) [inline nerds-catalog.html]: just write ./nerds-index.html

File: index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const feedbackLink = document.querySelector(".feedback-link");
```
Comment (universezzz) [inline index.js]: index.js -> script.js

File: nerds-catalog.html [hunk 4]
```diff
@@ -9,38 +9,43 @@
...
+              width="160"
+              height="65"
+            />
+          </a>
+        </div>
+        <nav class="header-navigation">
+          <ul>
+            <li><a href="#">Студия</a></li>
+            <li><a href="#">Клиенты</a></li>
+            <li><a href="../Nerds.web/nerds-catalog.html">Магазин</a></li>
```
Comment (universezzz) [inline nerds-catalog.html]: href="../Nerds.web/nerds-catalog.html" => href="./nerds-catalog.html"

File: nerds-catalog.html [hunk 5]
```diff
@@ -9,38 +9,43 @@
...
-          <li><a href="../Nerds.web/nerds-catalog.html">Магазин</a></li>
-          <li><a href="#">Контакты</a></li>
-        </ul>
-      </nav>
+    <header class="main-header">
+      <div class="header-container">
+        <div class="header-logo">
+          <a href="../Nerds.web/nerds-index.html" aria-label="Лого">
+            <img
+              src="../Nerds.web/img/index_img/index__img_svg/nerds-logo.svg"
```
Comment (universezzz) [inline nerds-catalog.html]: src="../Nerds.web/img/index_img/index__img_svg/nerds-logo.svg" =>   src="./img/index_img/index__img_svg/nerds-logo.svg"

File: nerds-catalog.html [hunk 6]
```diff
@@ -9,38 +9,43 @@
...
 
-      <div class="cart">
-        <a href="#"
-          ><img
-            src="../Nerds.web/img/index_img/index__img_svg/cart-icon.svg"
-            alt="Корзина"
-          />
-          Корзина</a
-        >
+        <div class="cart">
```
Comment (universezzz) [inline nerds-catalog.html]: div -> ul with li. In the future, different elements could be added.

File: nerds-catalog.html [hunk 7]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
+    <footer class="main-footer">
       <div class="map">
-        <img
-          src="../Nerds.web/img/index_img/map-marker.png"
-          alt="191186, Санкт-Петербург,ул. Б. Конюшенная, д. 19/8"
-        />
+        <div class="map-content">
+          <img
+            class="map-marker"
+            src="../Nerds.web/img/index_img/map-marker.png"
```
Comment (universezzz) [inline nerds-catalog.html]: wrong src

File: nerds-catalog.html [hunk 8]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
+            alt="ул. Б. Конюшенная, д. 19/8"
+            width="231"
```
Comment (universezzz) [inline nerds-catalog.html]: width/height write in css

File: nerds-catalog.html [hunk 9]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
+        </section>
 
-      <nav class="pagination">
-        <a href="#">1</a>
-        <a href="#">2</a>
-        <a href="#">3</a>
-        <a href="#">Следующая</a>
-      </nav>
+        <nav class="pagination">
+          <a href="#">1</a>
```
Comment (universezzz) [inline nerds-catalog.html]: ol->li->a

File: nerds-catalog.html [hunk 10]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
+              alt="Хэдер браузера"
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-kvast.jpg"
+              alt="Макет кваст"
+            />
+            <div class="product-item-hover">
+              <h4>КВАСТ</h4>
+              <p>Промо-сайт производителя крафтового кваса</p>
+              <div>
```
Comment (universezzz) [inline nerds-catalog.html]: for what did you use `<div>`?

File: nerds-catalog.html [hunk 11]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
-      </section>
+          <article class="product-item">
+            <h3 class="visually-hidden">Макет кваст</h3>
+            <img
+              class="browser-img"
+              src="../Nerds.web/img/catalog_img/catalog__img_svg/browser.svg"
+              alt="Хэдер браузера"
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-kvast.jpg"
```
Comment (universezzz) [inline nerds-catalog.html]: src

File: nerds-catalog.html [hunk 12]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
-					<p>Промо-сайт производителя крафтового кваса</p>
-					<div><a href="#" aria-label="Цена">9900 РУБ.</a></div>
-				</div> -->
-        </article>
-      </section>
+          <article class="product-item">
+            <h3 class="visually-hidden">Макет кваст</h3>
+            <img
+              class="browser-img"
+              src="../Nerds.web/img/catalog_img/catalog__img_svg/browser.svg"
```
Comment (universezzz) [inline nerds-catalog.html]: src

File: nerds-catalog.html [hunk 13]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
-        </article>
+          <article class="product-item">
+            <h3 class="visually-hidden">Макет A Plus</h3>
+            <img
+              class="browser-img"
+              src="../Nerds.web/img/catalog_img/catalog__img_svg/browser.svg"
+              alt="Хэдер браузера"
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-aplus.jpg"
```
Comment (universezzz) [inline nerds-catalog.html]: src

File: nerds-catalog.html [hunk 14]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
-					<h4>A PLUS</h4>
-					<p>Лендинг курсов повышения квалификации</p>
-					<div><a href="#" aria-label="Цена">9900 РУБ.</a></div>
-				</div> -->
-        </article>
+          <article class="product-item">
+            <h3 class="visually-hidden">Макет A Plus</h3>
+            <img
+              class="browser-img"
+              src="../Nerds.web/img/catalog_img/catalog__img_svg/browser.svg"
```
Comment (universezzz) [inline nerds-catalog.html]: src

File: nerds-catalog.html [hunk 15]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-aplus.jpg"
+              alt="Макет A Plus"
+            />
+            <div class="product-item-hover">
+              <h4>A PLUS</h4>
+              <p>Лендинг курсов повышения квалификации</p>
+              <div>
+                <a href="#" aria-label="Цена" class="button">9900 РУБ.</a>
```
Comment (universezzz) [inline nerds-catalog.html]: make a display: block for link instead of using div if it's possible

File: nerds-catalog.html (hunk 10 above)
Comment (universezzz) [reply nerds-catalog.html]: make a display: block for link instead of using div if it's possible

File: nerds-catalog.html [hunk 16]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
+              alt="Хэдер браузера"
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-barbershop.jpg"
+              alt="Макет Barbershop Borodonski"
+            />
+            <div class="product-item-hover">
+              <h4>BARBERSHOP</h4>
+              <p>Сайт салоны красоты. Для мужчин, да.</p>
+              <div>
```
Comment (universezzz) [inline nerds-catalog.html]: make a display: block for link instead of using div if it's possible

File: nerds-catalog.html [hunk 17]
```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
...
+              alt="Хэдер браузера"
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-sedona.jpg"
+              alt="Макет Sedona"
+            />
+            <div class="product-item-hover">
+              <h4>SEDONA</h4>
+              <p>Информационный сайт для туристов</p>
+              <div>
```
Comment (universezzz) [inline nerds-catalog.html]: make a display: block for link instead of using div if it's possible

File: index.css [hunk 18]
```diff
@@ -0,0 +1,920 @@
...
+
+  background-color: transparent;
+  opacity: 0.3;
+
+  cursor: pointer;
+}
+
+.feedback-modal .close-cross::before,
+.feedback-modal .close-cross::after {
+  content: " ";
```
Comment (universezzz) [inline index.css]: empty content

File: index.css [hunk 19]
```diff
@@ -0,0 +1,920 @@
...
+}
+
+.social-footer a {
+  display: block;
+
+  width: 78px;
+  height: 78px;
+  box-sizing: border-box;
+
+  display: flex;
```
Comment (universezzz) [inline index.css]: display: flex or display: block? You should decide it.

File: catalog.css [hunk 20]
```diff
@@ -0,0 +1,486 @@
...
+
+.header-container {
+  margin: 0 auto;
+  margin-bottom: 50px;
+  padding-bottom: 45px;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+	align-items: center;
```
Comment (universezzz) [inline catalog.css]: remove tag, don't forget about prettier

File: catalog.css [hunk 21]
```diff
@@ -0,0 +1,486 @@
...
+  background-image: url(../Nerds.web/img/index_img/index__img_svg/cart-icon.svg);
+  background-repeat: no-repeat;
+  background-position: 0 0;
+}
+
+.main-header h1 {
+  margin: 0;
+  padding: 0;
+
+  margin: 0 auto;
```
Comment (universezzz) [inline catalog.css]: margin: 0 or margin: 0 auto?

File: index.css [hunk 22]
```diff
@@ -0,0 +1,920 @@
...
+}
+
+.projects li:not(:first-child) {
+  padding-left: 50px;
+
+  position: relative;
+}
+
+.projects li:not(:first-child)::before {
+  content: " ";
```
Comment (universezzz) [inline index.css]: empty content

File: nerds-catalog.html (hunk 10 above)
Comment (Madesine) [reply nerds-catalog.html]: idk )

**Pull request merged**